### PR TITLE
Fix suggested route and hire rate and suggested route

### PR DESCRIPTION
**Pull request by ismailirahim**

### CODE REVIEW

**Review by fawadik — APPROVED** [review]
looks good

File: src/client/components/SelectedRouteContainer.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwicm9sZSI6InN1cGVydXNlciIsImlhdCI6MTU3MTIxNzY5MiwiZXhwIjoxNTcxMjUzNjkyfQ.GLBygWbn6nO6M-ly4ACX7wKefIXpq4wT6ocLUjzdV-M"
+        }
+      })
+      .then(data => console.log(data));
+  }
+
+  render() {
+    console.log(this.props.match);
+    return <div>hello</div>;
+  }
```
Comment (zkwsk) [inline src/client/components/SelectedRouteContainer.js:21]: I'm assuming this is work-in-progress.

File: src/server/api/controllers/suggested-routes.controller.js [hunk 2]
```diff
@@ -39,39 +39,66 @@ const getSuggestedRouteById = async id => {
...
+        "rpm",
+        "sequence_id"
+      );
+
+    return suggestedRoutes.map(item => {
+      item.waypoint = [];
+      waypoints.forEach(elem => {
+        if (item.id === elem.suggested_route_id) {
+          item.waypoint.push(elem);
+        }
```
Comment (zkwsk) [inline src/server/api/controllers/suggested-routes.controller.js:97]: Potentially use .filter for this.

**Review by zkwsk — APPROVED** [review]
A few small potential improvements, but let's get the ball rolling 🎈

**Pull request merged**